### DISCUSSION TITLE
Invite mail not sent

### DISCUSSION THREAD

**Discussion by taiki14175** [AntlerVC/firetable#1448] [Support / Q&A]
I'm trying invite a member but I can't get any email.
How can I invite other users?

I also tried this.
https://docs.rowy.io/how-to/roles?set-user-roles-tabs=admin-sdk#set-user-roles-tabs

Thank you for reading.

**Comment by Chadha93** [comment]
Hi @taiki14175, Did you try inviting a member to your workspace using user management? As shown in this video - https://www.youtube.com/watch?v=DmmI0L9BuUE&t=136s.

As you have mentioned, you have tried this the admin SDK approach, could you please mention the issue or error you are facing in this approach - https://docs.rowy.io/how-to/roles?set-user-roles-tabs=admin-sdk#set-user-roles-tabs

**Comment by taiki14175** [comment]
Hi Chadha93, Thank you for your replying.
I checked the video.
Yes, I invited from user management page. 
I didn't make new role at that time. 
I just set ADMIN role and new member email address.

I retried admin SDK approach.
I run a Node.js script on Firebase Authentication users, but the program ends without any errors.
 
I apologize for the lack of information.
What kind of things could be considered?

**Reply by Chadha93** [reply]
Hi @taiki14175,

Could you please try assigning the role at the time you invite, it from the user management page? The 2nd SDK approach seems an issue with the code, what was the response of the execution?

**Reply by Chadha93** [reply]
Closing the discussion, feel free to reach out on Discord, if the issue still exists.
